src/engraving/rendering/dev/slurtielayout.cpp

```diff
@@ -2490,22 +2490,22 @@ void SlurTieLayout::layoutSegment(SlurSegment* item, LayoutContext& ctx, const P
     computeBezier(item);
 }
 
-void SlurTieLayout::computeMidThickness(SlurTieSegment* slurTieSeg, double slurTieLengthInSp)
+void SlurTieLayout::computeMidThickness(TieSegment* tieSeg, double tieLengthInSp)
```

Comment:
This method was written like that for the specific purpose of being used by both slurs and ties (that is done in #20481 which isn't yet merged). Please wait for that one to be merged, then readapt these changes

Reply:
Update: #20481 has been merged now :)

Reply:
If slurs now use this method, won't they be affected by `Sid::MinTieLength`, a property implied to affect only ties?